test/simplifyExpression/fractionsSearch/cancelLikeTerms.test.js

```diff
@@ -23,6 +23,7 @@ describe('cancel like terms', function () {
     ['-x / -x', '1'],
     ['2/ (4x)', '1 / (2x)'],
     ['2/ (4x^2)', '1 / (2x^2)'],
+    ['2 a / a', '2'],
```

Comment:
Can you add the cases that errored somewhere? I guess they might not go into this test because we _don't_ want to cancel terms for them, but let's add them as test cases somewhere else.